### PR TITLE
Write file output to disk option

### PR DESCRIPTION
**Pull request by Crabzmatic**
Add option to write output value (from get command) to .jpeg file to disk.
Naming is {key}.jpg

### CODE REVIEW

**Comment by liderman** [comment]
I think extending the get method is a bad idea.
Each method must perform one action.
Better to add a new command "export":
```
export `KEY` `FILENAME`
```

**Comment by Crabzmatic** [comment]
> I think extending the get method is a bad idea.
> Each method must perform one action.
> Better to add a new command "export":
> 
> ```
> export `KEY` `FILENAME`
> ```

I made a new export command and changed the version to `0.3.0`.

File: README.md [hunk 1]
```diff
@@ -89,6 +89,13 @@ Display value by key.
  * `KEY` - The key
  * `FORMAT` - Data Display Format (Optional)
 
+### export
+> export `KEY` `FILENAME`
+
+Write value by key to .jpg file with specified filename.
```
Comment (liderman) [inline README.md]: Why only the extension ".jpg"?
user must be able to specify any file name or path.
Please change to:
```
Write value by key to file with specified filename or path.
```

File: commands/exportCommand.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+// This software provides a console interface to leveldb.
+
+package commands
+
+import (
+	"io/ioutil"
+)
+
+// The command exports a value for the selected key
+// to .jpeg file with specified filename.
```
Comment (liderman) [inline commands/exportCommand.go]:
```
to file with specified filename.
```

File: commands/exportCommand.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	if key == "" {
+		return AppError(ErrKeyIsEmpty)
+	}
+
+	value, err := dbh.Get([]byte(key), nil)
+	if err != nil {
+		return AppError(ErrKeyNotFound)
+	}
+
+	fileErr := ioutil.WriteFile(fileName+".jpg", value, 0644)
```
Comment (liderman) [inline commands/exportCommand.go]:
```go
fileErr := ioutil.WriteFile(fileName, value, 0644)
```

File: main.go [hunk 4]
```diff
@@ -141,6 +143,15 @@ func main() {
 
 			fmt.Println(commands.Get(args[1], format))
 			break
+		//Command export
+		case args[0] == "export":
+			if len(args) < 3 || len(args) > 3 {
```
Comment (liderman) [inline main.go]: The code could be simpler:
```go
if len(args) != 3 {
```

Comment (Crabzmatic) [reply main.go]: simplified

File: commands/exportCommand.go (hunk 3 above)
Comment (Crabzmatic) [reply commands/exportCommand.go]: done

File: commands/exportCommand.go (hunk 2 above)
Comment (Crabzmatic) [reply commands/exportCommand.go]: done

File: README.md (hunk 1 above)
Comment (Crabzmatic) [reply README.md]: done

**Comment by liderman** [comment]
@Crabzmatic Thanks! Good job

**Pull request merged**